Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.yml

```diff
@@ -0,0 +1,211 @@
+commonfields:
+  id: ShadowxSOCAI
+  version: -1
+name: ShadowxSOCAI
+display: ShadowX SOCAI
+category: Analytics & SIEM
+description: ShadowX AI plugin that analyzes security logs and returns response, recommendation, risk/result, and prediction score. Supports API Key (JSON /Api/ endpoints) and cookie (HTML) modes.
+fromversion: 6.10.0
+
+configuration:
+- display: Server URL (e.g., https://app.shadowx.ai)
```

Comment:
The parameter definitions in the `configuration` section are structured unconventionally. For consistency with XSOAR standards and better readability, please start each parameter definition with the `name` key, followed by `display` and other keys.